src/assets/i18n/en-us.json

```diff
@@ -52,14 +52,15 @@
   "logout": "Log Out",
   "tags": "Tags",
   "all": "All",
-  "viewAll": "View All",
+  "viewAll": "open more...",
+  "moreInformation": "More Information",
   "mediaId": "Media ID",
   "media": "Media",
   "selectContact": "Select Contact",
   "inviteFriend": "Invite Friend",
   "invitationEmail": "Invitation Email",
   "sendPostCapture": "Send PostCapture",
-  "previewPostCapture": "Preview PostCapture",
+  "previewPostCapture": "Preview",
```

Comment:
Remove the property `"preview"`. The properties in root is for title which is capitalized to its key.